source/Outfit.cpp

```diff
@@ -261,6 +302,13 @@ const string &Outfit::Name() const
 
 
 
+const string &Outfit::DisplayName() const
```

Comment:
The convention is `Name()` returns the display name and `TrueName()` returns the name in the data files. We should stay consistent :)